sormas-api/src/main/java/de/symeda/sormas/api/symptoms/SymptomsHelper.java

```diff
@@ -228,13 +235,9 @@ public static void updateSymptoms(SymptomsDto sourceSymptoms, SymptomsDto target
 					}
 				} else if (pd.getReadMethod().getReturnType() == Boolean.class) {
 					// Booleans are carried over when they are TRUE
-					if (pd.getName().equals(SymptomsDto.SYMPTOMATIC)) {
-						continue;
-					} else {
-						Boolean result = (Boolean) pd.getReadMethod().invoke(sourceSymptoms);
-						if (Boolean.TRUE.equals(result)) {
-							pd.getWriteMethod().invoke(targetSymptoms, result);
-						}
+					YesNoUnknown result = (YesNoUnknown) pd.getReadMethod().invoke(sourceSymptoms);
```

Comment:
Revert this change. This doesn't make sense, you're asking whether the return type is Boolean and then process the field as if it were a YesNoUnknown field. Please just restore the former code, get rid of the if condition and only use the code in the else block.